modules/compliance-operator-rosa-installation.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/co-management/compliance-operator-installation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-compliance-operator-rosa_{context}"]
+= Installing the Compliance Operator on ROSA hosted control planes (HCP)
+
+As of the Compliance Operator 1.5.0 release, the Operator is tested against {product-rosa} using {hcp-capital}.
+
+{product-rosa} {hcp-capital} clusters have restricted access to the control plane, which is managed by Red Hat. By default, the Compliance Operator will schedule to nodes within the `master` node pool, which is not available in {product-rosa} {hcp-capital} installations. This requires you to configure the `Subscription` object in a way that allows the Operator to schedule on available node pools. This step is necessary for a successful installation on {product-rosa} {hcp-capital} clusters.
```

Comment:
Per the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces) there should be a non breaking space in between Red and Hat:
```suggestion
{product-rosa} {hcp-capital} clusters have restricted access to the control plane, which is managed by Red{nbsp}Hat. By default, the Compliance Operator will schedule to nodes within the `master` node pool, which is not available in {product-rosa} {hcp-capital} installations. This requires you to configure the `Subscription` object in a way that allows the Operator to schedule on available node pools. This step is necessary for a successful installation on {product-rosa} {hcp-capital} clusters.
```

Reply:
Thank you, done!